### ISSUE TITLE
when is the new release 4.12. available on addons.mozilla.org?

### ISSUE THREAD

**Issue by s-light** [greasemonkey/greasemonkey#3179]
just stumble over the 4.12 release announcement on the  [website says](https://www.greasespot.net/)
> Saturday, December 02, 2023
> [Greasemonkey 4.12 Release](https://www.greasespot.net/2023/12/greasemonkey-412-release.html)

that is a very good news :-)

but sadly i can not find this update on 
https://addons.mozilla.org/en-US/firefox/addon/greasemonkey/
there is only the *old* version shown:
```
Version
    4.11
Size
    537.08 KB
Last updated
    3 years ago (Jan 28, 2021)
```

here in the repository is also no tag for the 4.12 Release ?!
https://github.com/greasemonkey/greasemonkey/tags

the [milestone linked from the hompeage](https://github.com/greasemonkey/greasemonkey/milestone/94?closed=1)
is done and closed... and the [last commit](https://github.com/greasemonkey/greasemonkey/commit/c0b3b542074b48793fb24023da114cfeb643ec95) is the version bump...

so do i just missing something else?! 
or do i just need to wait a while till the update is visible on addons.mozilla.org ?

**Comment by arantius** [comment]
Thanks for asking.  Turns out (much like 4.10) this was mistakenly set to
"self hosted" (because of IMO a flaw in addons.mozilla.org).  So I'll have
to upload it again.  Which I've just done, with the same ".0" version
addition like 4.10 had.  Standard AMO review window caveats apply.

**Comment by krystian3w** [comment]
IMO fixed, I see 4.12:

![obraz](https://github.com/greasemonkey/greasemonkey/assets/35370833/9f962dfb-4c6d-434c-9c4b-20e4c304e30d)

[greasemonkey-4.12.0.zip](https://github.com/greasemonkey/greasemonkey/files/13855111/greasemonkey-4.12.0.zip)

**Comment by arantius** [comment]
Yes thank you it is indeed fixed.
